### PR TITLE
Docs: Update docs for enterprise reporting

### PR DESCRIPTION
**Pull request by undef1nd**
**What this PR does / why we need it**:
Updates docs to describe recent changes in reporting scheduler.

- [x] Add a new screenshot of the report settings page and scheduler  https://github.com/grafana/website/pull/5439

**Special notes for your reviewer**:

### CODE REVIEW

**Review by osg-grafana — CHANGES_REQUESTED** [review]

File: docs/sources/enterprise/reporting.md [hunk 1]
```diff
@@ -106,48 +106,26 @@ When the CSV file is generated, it is temporarily written to the `csv` folder in
 A background job runs every 10 minutes and removes temporary CSV files. You can configure how long a CSV file should be stored before being removed by configuring the [temp-data-lifetime]({{< relref "../administration/configuration/#temp-data-lifetime" >}}) setting. This setting also affects how long a renderer PNG file should be stored.
 
 ### Scheduling
+> Note: Scheduler has been significantly changed in Grafana Enterprise v8.1.
 
-Scheduled reports can be sent on a monthly, weekly, daily, or hourly basis. You may also disable scheduling for when you either want to send it via the API.
+Scheduled reports can be sent once or repeated on an hourly, daily, weekly and monthly basis, or at custom intervals.  You can also disable scheduling by selecting **Never** if you want to send the report via the API.
```
Comment (Clarity-89) [inline docs/sources/enterprise/reporting.md:111]:
```suggestion
Scheduled reports can be sent once or repeated on an hourly, daily, weekly and monthly basis, or at custom intervals.  You can also disable scheduling by selecting **Never** or if you want to send the report via the API.
```

**Review by Clarity-89 — COMMENTED** [review]
Looks good, but I'll leave the final approval to the docs squad.

File: docs/sources/enterprise/reporting.md (hunk 1 above)
Comment (undef1nd) [reply docs/sources/enterprise/reporting.md:111]: Not sure about this. I meant to say that "Never" can be used to disable scheduling, for example, when report distribution needs to be triggered via API.

Comment (Clarity-89) [reply docs/sources/enterprise/reporting.md:111]: What I meant is that in the current state it reads as if you'd only set schedule to Never to send reports via API, but sometimes you'd do that just not to send a report. I might have misunderstood the sentence though.

Comment (undef1nd) [reply docs/sources/enterprise/reporting.md:111]: You're right. Thank you!

File: docs/sources/enterprise/reporting.md [hunk 2]
```diff
@@ -106,48 +106,26 @@ When the CSV file is generated, it is temporarily written to the `csv` folder in
 A background job runs every 10 minutes and removes temporary CSV files. You can configure how long a CSV file should be stored before being removed by configuring the [temp-data-lifetime]({{< relref "../administration/configuration/#temp-data-lifetime" >}}) setting. This setting also affects how long a renderer PNG file should be stored.
 
 ### Scheduling
+> Note: Scheduler has been significantly changed in Grafana Enterprise v8.1.
 
-Scheduled reports can be sent on a monthly, weekly, daily, or hourly basis. You may also disable scheduling for when you either want to send it via the API.
+Scheduled reports can be sent once or repeated on an hourly, daily, weekly and monthly basis, or at custom intervals. You can also disable scheduling by selecting **Never**: for example, if you want to send the report via the API.
```
Comment (osg-grafana) [inline docs/sources/enterprise/reporting.md:111]:
```suggestion
Scheduled reports can be sent once or repeatedly on an hourly, daily, weekly, or monthly basis, or at custom intervals. You can also disable scheduling by selecting **Never**: for example, if you want to send the report via the API.
```

File: docs/sources/enterprise/reporting.md [hunk 3]
```diff
@@ -106,48 +106,26 @@ When the CSV file is generated, it is temporarily written to the `csv` folder in
...
 
-Hourly reports are generated once per hour. All fields are required.
+**Send now or schedule for later**
 
-- **At minute -** The number of minutes after full hour when the report should be generated.
-- **Time zone -** Time zone to determine the offset of the full hour. Does not currently change the time in the rendered report.
+- **Send now** sends the report immediately after you save it. To stop sending the report at some point in the future, add an end date. If you leave the end date empty, the report is sent out indefinitely.
 
-#### Daily
+- **Send later** schedules a report for a later date, so you must set a start date and time. If you leave the end date empty, the report is sent out indefinitely.
```
Comment (osg-grafana) [inline docs/sources/enterprise/reporting.md:120]:
```suggestion
- **Send later** schedules a report for a later date, if you set a start date and time. If you leave the end date empty, the report is sent out indefinitely.
```
The sentences cause an error in logic.

File: docs/sources/enterprise/reporting.md [hunk 4]
```diff
@@ -106,48 +106,26 @@ When the CSV file is generated, it is temporarily written to the `csv` folder in
...
 
-Daily reports are generated once per day. All fields are required.
+**Send only from Monday to Friday**
 
-- **Time -** Time the report is sent, in 24-hour format.
-- **Time zone -** Time zone for the **Time** field.
+You can choose to send some reports only from Monday to Friday. This applies to reports that have an hourly or daily frequency.
```
Comment (osg-grafana) [inline docs/sources/enterprise/reporting.md:124]:
```suggestion
For reports that have an hourly or daily frequency, you can choose to send them only from Monday to Friday.
```

**Review by osg-grafana — APPROVED** [review]
Unblocking; please incorporate feedback.

File: docs/sources/enterprise/reporting.md (hunk 3 above)
Comment (undef1nd) [reply docs/sources/enterprise/reporting.md:120]: I meant to say that because this options sends report later on a date, the date and time are required.

**Comment by osg-grafana** [comment]
@undef1nd, I might be wrong about the backport label that I just applied. Please vet that to make sure.

**Pull request merged**